src/client/pfs_v2.go

```diff
@@ -65,22 +78,22 @@ func (c APIClient) NewFileOperationClientV2(repo, commit string) (_ *FileOperati
 }
 
 // PutTar puts a tar stream into PFS.
-func (foc *FileOperationClient) PutTar(r io.Reader, tag ...string) error {
+func (foc *FileOperationClient) PutTar(r io.Reader, overwrite bool, tag ...string) error {
 	return foc.maybeError(func() error {
+		ptr := &pfs.PutTarRequestV2{Overwrite: overwrite}
 		if len(tag) > 0 {
```

Comment:
This might be cleaner as a switch.

Reply:
I prefer the current pattern for consistency reasons. Most variadic functions in our codebase capture the handling of the optional arguments in a single block with the check len(args) > 0.